src/AzureAuth/Commands/CommandAzureAuth.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Authentication.AzureAuth.Commands
     /// <summary>
     /// The command main class parses commands and dispatches to the corresponding methods.
     /// </summary>
-    [Command(Name = "azureauth", Description = "A CLI interface to MSAL (Microsoft Authentication Library).")]
+    [Command(Name = "azureauth", Description = "A command line interface to MSAL (Microsoft Authentication Library).")]

```

Comment:
Should we also cover the ability to generate PATs in the text?

Reply:
That's covered by one of the subcommands. A whole overhaul of our help text might be worthwhile, but I think that should be a separate PR. 😆